alembic.ini

```diff
@@ -0,0 +1,116 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = alembic/
```

Comment:
Better call the directory migrations?

Reply:
Yeah we can call it that so that people who aren't familiar with what _alembic_ means know what this directory is. But most of the projects in Canonical call it "alembic". 
[hiring.canonical.com](https://github.com/canonical/hiring.canonical.com)
[workspaceengineering-bot](https://github.com/canonical/workspaceengineering-bot/)
[myworkplace.canonical.com](https://github.com/canonical/myworkplace.canonical.com)

Reply:
Right, let's keep it that way then.